### DISCUSSION TITLE
Cannot style LED color

### DISCUSSION THREAD

**Discussion by matsekberg** [HASwitchPlate/openHASP#718] [Q&A]
### Perform all steps below and tick them with [x]
- [x] Check the related part of the [Documentation](https://www.openhasp.com/)
- [x] Update openHASP to the latest version
- [x] Reproduce the issue and describe all steps

### Describe the bug
I try to change the color of the LED object 

### To Reproduce

Version 0.7.0

```json
{"page":1,"id":0,"comment":"---------- SALON ---------- 320x480", "swipe":1,"bg_color":"black", "bg_grad_color":"black"}

{"id":11,"obj":"led","x":10,"y":60,"w":20,"h":20,"color":"#FF0000"}
{"id":12,"obj":"led","x":60,"y":60,"w":20,"h":20,"color":"#FF0000","val":200}
{"id":13,"obj":"led","x":110,"y":60,"w":20,"h":20,"color":"#FF0000","val":10}

{"id":21,"obj":"led","x":10,"y":100,"w":20,"h":20,"color":"#00FF00"}
{"id":22,"obj":"led","x":60,"y":100,"w":20,"h":20,"color":"#00FF00","val":200}
{"id":23,"obj":"led","x":110,"y":100,"w":20,"h":20,"color":"#00FF00","val":10}

{"id":21,"obj":"led","x":10,"y":140,"w":20,"h":20,"color":"#FFFF88"}
{"id":22,"obj":"led","x":60,"y":140,"w":20,"h":20,"color":"#FFFF88","val":200}
{"id":23,"obj":"led","x":110,"y":140,"w":20,"h":20,"color":"#FFFF88","val":10}
```

### Expected behavior

![IMG_5A7A77D47515-1](https://github.com/HASwitchPlate/openHASP/assets/4611611/5cf06c3b-d454-40e0-83a6-46a2183b5a18)

### Screenshots or video

![screenshot (1) Liten](https://github.com/HASwitchPlate/openHASP/assets/4611611/25b9a6b6-c887-4c4a-8bc8-2f4cb24f8b05)

**Comment by sfromis** [comment]
Looks like you copied an [example from HASPmota](https://tasmota.github.io/docs/HASPmota/#led), without being clear about this. While this is using a `jsonl` format "heavily inspired" by openHASP, it is not a guide about how to do things in openHASP.

**Comment by matsekberg** [comment]
I agree, but its an example, one of many i have tried.

**Comment by marsman7** [comment]
Take a look at the IDs of the objects, there are duplicates.

**Comment by fvanroie** [comment]
This works for me:

```json
{"page": 3,"id": 15,"obj": "led","x": 10,"y": 10,"w": 30,"h": 30,"bg_color":"red","border_color":"yellow","shadow_color":"orange"}
```

![image](https://github.com/HASwitchPlate/openHASP/assets/15969459/02535b80-799e-4d3c-9925-c2cf8073836e)

Remember, openHASP != haspmota, they are _very different_ below the covers.

**Reply by matsekberg** [reply]
Ok I tried now with bg_color and it works, strange, I have tried that before but must have messed up somehow.
Wouldnt it be more correct to have bg_color10 style the LED?
